### DISCUSSION TITLE
posts/authelia-traefik/

### DISCUSSION THREAD

**Discussion by giscus** [techno-tim/techno-tim.github.io#273] [Comments]
# posts/authelia-traefik/

Authelia is an open source Single Sign On and 2FA companion for reverse proxies.It helps you secure your endpoints with single factor and 2 factor auth.It works with Nginx, Traefik, and HA proxy.Today, we’ll configure Authelia with Portainer and Traefik and have 2 Factor up and running with brute force protection!

https://technotim.live/posts/authelia-traefik/

**Comment by Ahosking76** [comment]
It is not working for me.  When I try to go to the login URL for authelia (https://auth.usahosking.com) , I get "404 page not found".  I have the DNS entry through cloudflare and it looks like the certs are working.  I must be missing something in traefik?
